### PR TITLE
SC Election 2024 announcement

### PR DESCRIPTION
**Pull request by infinisil**
:rocket:

Ping @thilobillerbeck

### CODE REVIEW

File: src/content/blog/announcements/2024-09-16_sc-election-2024.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Timeline
+
+- 2024-09-29 Sun: Deadline for nominations
+- 2024-10-01 Tue: Deadline for endorsements
+- 2024-10-03 Thu: Deadline for candidate questions
+- 2024-10-06 Sun: Deadline for candidate answers,
+  submitting voter exceptions and updating voter email addresses
+- 2024-10-07 Mon: Voting starts
+- 2024-10-20 Sun: Voting ends
+- 2024-10-23 Wed: Results announced
```
Comment (alyssais) [inline src/content/blog/announcements/2024-09-16_sc-election-2024.md:64]: Should this also mention deadline for applying for exceptions?

**Review by thilobillerbeck — APPROVED** [review]

**Comment by thilobillerbeck** [comment]
Could you please format your changes with `npm format` so CI does not fail. Then I'll merge

File: src/content/blog/announcements/2024-09-16_sc-election-2024.md (hunk 1 above)
Comment (alyssais) [reply src/content/blog/announcements/2024-09-16_sc-election-2024.md:64]: Wait it does, nm.

**Comment by infinisil** [comment]
@thilobillerbeck Done!

**Pull request merged**